app/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 <head>
 <meta charset="utf-8">
-<meta http-equiv="Content-Security-Policy" content="default-src 'self' ; style-src 'self' 'unsafe-inline' ; connect-src * ; object-src 'none' ; referrer no-referrer ;">
+<meta http-equiv="Content-Security-Policy" content="default-src 'self' ; style-src 'self' 'unsafe-inline' ; connect-src * ; object-src 'none' ; img-src * ; referrer no-referrer ;">
```

Comment:
The CSP is static, so `img-src *` is good.
You may also want to alter the `media-src` directive if you want to stream video from the box.